users/konstantin/post_config.h

```diff
@@ -0,0 +1,6 @@
+#pragma once
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Do I need to do this for every header in my userspace?

Reply:
ideally, for every c and h file, yeah. 

But that can be a "later" thing.

Reply:
No problem, I'll add them now.